src/integrationTest/java/uk/gov/hmcts/reform/sscs/controllers/ServiceHearingsControllerTest.java

```diff
@@ -191,43 +145,18 @@ void testPostRequestServiceLinkedCases() throws Exception {
     void testPostRequestServiceLinkedCases_badCaseID() throws Exception {
 
         mockMvc.perform(post(SERVICE_LINKED_CASES_URL)
-                        .header(SERVICE_AUTHORISATION_HEADER, AUTHORIZATION)
                         .param(CASE_REFERENCE,BAD_CASE_ID)
                         .param(CASE_REFERENCE_PARAM, String.valueOf(HEARING_ID)))
                 .andDo(print())
                 .andExpect(status().isBadRequest());
     }
 
-    @DisplayName("When Authorization is incorrectly formatted should return a with 400 response code")
-    @Test
-    void testPostRequestServiceLinkedCases_badAuthHeader() throws Exception {
-
-        mockMvc.perform(post(SERVICE_LINKED_CASES_URL)
-                        .header(SERVICE_AUTHORISATION_HEADER, BAD_AUTHORIZATION)
-                        .param(CASE_REFERENCE, String.valueOf(CASE_ID))
-                        .param(CASE_REFERENCE_PARAM, String.valueOf(HEARING_ID)))
-                .andDo(print())
-                .andExpect(status().isBadRequest());
-    }
-
-    @DisplayName("When not Authorized should return a with 403 response code")
-    @Test
-    void testPostRequestServiceLinkedCases_unauthorised() throws Exception {
-
-        mockMvc.perform(post(SERVICE_LINKED_CASES_URL)
-                        .header(SERVICE_AUTHORISATION_HEADER, NOT_AUTHORIZATION)
-                        .param(CASE_REFERENCE, String.valueOf(CASE_ID))
-                        .param(CASE_REFERENCE_PARAM, String.valueOf(HEARING_ID)))
-                .andDo(print())
-                .andExpect(status().isForbidden());
-    }
```

Comment:
Was this intentional to remove unit tests?

Reply:
Yes, those were testing the behaviour of the authorisationService which was removed in favour of the filter change